### PR TITLE
Ironrot reagent

### PR DESCRIPTION
**Pull request by RealestEstate**
New reagent: Ironrot. Created by mixing iron powder, radium, and alpha amanatin. Rots walls and damages silicon.

:cl:
 * rscadd: New reagent: Ironrot. Created by mixing iron powder, radium, and alpha amanatin. Rots walls and damages silicon it comes in contact with.

### CODE REVIEW

**Comment by Probe1** [comment]
You are a content saint.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -7389,6 +7389,42 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 		return TRUE
 	O.apply_luminol()
 
+/datum/reagent/ironrot
+	name = "Ironrot"
+	id = IRONROT
+	description = "A mutated fungal compound that causes rapid rotting in iron infrastructures."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#005200" //moldy green
```
Comment (Probe1) [inline code/modules/reagents/Chemistry-Reagents.dm:7397]: 👌 thats a quality comment.

**Comment by Dacendeth** [comment]
is this pacid smoke, but for borgs?

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (RealestEstate) [reply code/modules/reagents/Chemistry-Reagents.dm:7397]: thanks dad

**Comment by RealestEstate** [comment]
@Dacendeth yeah buddy

**Comment by Probe1** [comment]
Whats the yield a plant will deliver AMANATIN ?  Looking at the details, that borg damage is pretty dangerous.  If people could mass farm loads of this stuff it'd be a very _potent_ weapon.

 I'm not super fussed if the balance isn't 100% on day 1 as long as we're quick to fine tune it but I don't want to merge the silicon holocaust.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -7389,6 +7389,42 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
...
+
+/datum/reagent/ironrot/reaction_turf(var/turf/simulated/T, var/volume)
+	if(..())
+		return 1
+
+	if(volume >= 5 && T.can_thermite)
+		T:rot()
```
Comment (boy2mantwicethefam) [inline code/modules/reagents/Chemistry-Reagents.dm:7404]: ":"?

**Comment by RealestEstate** [comment]
@Probe1 so from what I understood looking at the code, the most amanatin a destroying angel can have is 3 at 150 potency if I understood correctly. that would mean 3+3+3 = 9units of iron rot per 150 potency angels, otherwise it's 6. I am totally open to scaling down the damage if you think it'd be more appropriate

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 2 above)
Comment (RealestEstate) [reply code/modules/reagents/Chemistry-Reagents.dm:7404]: it werks...
this is how the rotting random event writes it

Comment (RealestEstate) [reply code/modules/reagents/Chemistry-Reagents.dm:7404]: https://github.com/vgstation-coders/vgstation13/blob/89881f68efcc7939ba1b7e83e29c9636b7402d74/code/modules/events/wallrot.dm#L32

**Comment by RealestEstate** [comment]
as for yield, it's piss easy to get any plant to 5 yield using grass dev, so 5 times 9 would be 45 per harvest. I'll be honest I only tested the damage in smoke form, I didn't test using a chem sprayer or splashing borgs with a bucket.

**Comment by Probe1** [comment]
Have a looksy look at the other two that you listed and report back.  If it's going to be _balance changing_ the PR needs the tag.

**Comment by RealestEstate** [comment]
@Probe1 so apparently you can't splash beakers or buckets over borgs.
with a fire extinguisher it looked like I was doing about 10 damage (5 burn 5 brute) per spray. chemsprayer did about 5 damage per 5u per shot. keep in mind if anyone was using it with a sprayer they would probably dillute it since the quantity doesn't really have a factor on the damage. sprayers are weird but that's not my fault.

**Comment by Probe1** [comment]
It's probably fine then and if it's not I will fall on my sword

**Comment by ghost** [comment]
the silicon metaclub will remember this

**Comment by FudgePucker** [comment]
Polyacid is easy to make and fucking DESTROYS everything in the game other than silicons. What about smoke? Does it reck cyborgs?

**Comment by RealestEstate** [comment]
using it in smoke is probably the only reliable way to reck a borg, and even then assuming the borg stays stationary in the smoke for the entire duration he'll take only about 250 or so damage from a total healthpool of 300 iirc (though he'll be in really bad shape at that point and may as well be considered dead).

**Comment by SonixApache** [comment]
add a random event mushroom that has this in it for extra memes

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 2 above)
Comment (jknpj) [reply code/modules/reagents/Chemistry-Reagents.dm:7404]: Is this the legendary runtime operator?

Comment (jknpj) [reply code/modules/reagents/Chemistry-Reagents.dm:7404]: No, really, don't use it. Typecast the wall so you can call rot() directly

**Pull request merged**